cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,36 @@
+/* eslint-disable max-len */
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('Should be fill firstName', () => {
+    const name = 'Roma';
+    const lastName = 'Mykytiuk';
+    const email = 'mykytiuk@gmail.com';
+    const curentAdres = Math.floor(Math.random() * 100);
+    cy.get('#firstName').type(name);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('[for="gender-radio-1"]').click();
+    cy.get('#userNumber').type('123456789');
+    cy.get('#dateOfBirthInput').type('{enter}');
```

Comment:
The date of birth field is being submitted without any specific value. Consider setting a specific date to ensure the field is filled correctly and to verify its functionality.